### DISCUSSION TITLE
Welcome to cloud189 Discussions!

### DISCUSSION THREAD

**Discussion by Aruelius** [Aruelius/cloud189#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AAlexDing** [comment]
看到大佬在issue里展示的挂载演示了，很强大，不知道什么时候可以上线呢？

**Reply by Aruelius** [reply]
比较忙，加上没啥动力写，当时这个挂载遇到的问题也挺多的，对于 Python 来讲，没有一个成熟的挂载程序，网上光 fuse 包都一大堆，甚至不知道装哪个，装好了有几个也运行不了，最后翻出一个别人写的示例，一个个函数改，才有了预览图的样子。。。

**Reply by AAlexDing** [reply]
很强啦！祝事业顺利！
